### ISSUE TITLE
Imoen incapable of unlocking Charname's cage at the beginning

### ISSUE THREAD

**Issue by Hejtan** [Holic75/Baldurs-gate-dnd-3.5#16]
At the very beginning of Baldur's Gate 2, when Imoen enters the room and is about to talk with Charname and unlock his/her cage, she stops right next to the cage and starts "dancing", as if constantly trying and failing to interact with cage or something. This entirely blocks the game and makes it impossible to even get to the first dialogue.

**Comment by mathrud** [comment]
Same here. Is there any workaround?

**Comment by MaciejReszotnik** [comment]
Yeah, sam problem here as well. Mayhap this is the issue with the latest patch to the game?

**Comment by MaciejReszotnik** [comment]
The quick fix is to quickly hit Esc key once the cutscene starts. This let's you proceed.
